build.gradle

```diff
@@ -267,7 +267,7 @@ Properties:
   -Pmemory=[4g|8g|...]
   -Pnum_instances=[NUM_INSTANCES]
   -Pnexus=[NEXUS_URL] (NEXUS_URL is optional)
-  -POS=[centos-7|fedora-35|debian-10|ubuntu-18.04|opensuse-42.3|openeuler-22.03]
+  -POS=[centos-7|fedora-35|ubuntu-18.04|opensuse-42.3|openeuler-22.03]
```

Comment:
Replacing it with `debian-11` could be better than removing it. Since the description is outdated and not exhaustive, we can fix it in dedicated PR later.

Reply:
@iwasakims thanks for your advice, I will fix it